src/tokenOperations/codec.ts

```diff
@@ -0,0 +1,40 @@
+import BigNumber from "bignumber.js";
+import { Address } from "../address";
+import { IAddress } from "../interface";
+import * as contractsCodecUtils from "../smartcontracts/codec/utils";
+import * as codecUtils from "../utils.codec";
+
+export function stringToBuffer(value: string): Buffer {
```

Comment:
Would be nice to have some tests for all these functions to ensure that the next changes will not affect the expected output

Reply:
Added.